### ISSUE TITLE
Feedback on linkedin task issue #231 from JS3

### ISSUE THREAD

**Issue by SallyMcGrath** [CodeYourFuture/PD-curriculum#64]
**What type of request is it**
- [ ] New content
- [x] Update/additional content

https://github.com/CodeYourFuture/Module-JS3/issues/231

**Why do you want to suggest this update?**

Could this issue be set to stretch or some alternative be given for accessibility? Some of our trainees cannot create a social media presence for their own safety. This is a small group but not super rare.

In previous LinkedIn requirements, they have been able to use a different name and a generated profile image.
